app/src/main/java/com/rob729/quiethours/util/AppConstants.kt

```diff
@@ -9,4 +9,5 @@ object AppConstants {
     val FIRST_BOOT = "FIRST_BOOT"
     val NIGHT_MODE = "nightMode"
     val RINGTONE_MODE = "RINGTONE_MODE"
+    val APP_INTRO_CHECK = "isIntroduced"
```

Comment:
Make it val APP_INTRO_CHECK = "APP_INTRO_CHECK"

Reply:
ok